exercises/perfect-numbers/generator/generator.jl

```diff
@@ -0,0 +1,23 @@
+# canonical version = 1.3.0
+using Test
+
+using LazyJSON
```

Comment:
If you introduce new generator dependencies, please add them to the environment in [`generators/`](https://github.com/exercism/julia/tree/master/generators). Otherwise it will be harder to reproduce a working environment later on when running them again.